armi/reactor/tests/test_blocks.py

```diff
@@ -1118,20 +1118,6 @@ def test102_setPitch(self):
             self.Block.getComponent(Flags.INTERCOOLANT).getDimension("op"), pitch
         )
 
-    def test_UnshapedGetPitch(self):
-        """
-        Test that a homogenous block can be created with a specific pitch.
-        This functionality is necessary for making simple homogenous reactors.
```

Comment:
This seemed pretty hacky/unneeded. You can always make it a hex shape or whatever shape you expect your homog reactor to be made of. Fundementally UnshapedComponent having an outer pitch is really smelling, and if none of our testing needs it I think we should not support that feature.

Reply:
This seems to have been necessary for when people wanted to perform basic benchmark problems that only defined fundamental volume fractions. In these scenarios, if you have 25% coolant, 25% steel, and 50% fuel, you don't usually want to have to back-calculate some fake geometry that would have those fractions. This particular test is holding up that capability. 

If we want to remove it entirely we should also remove op from the unshaped component constructor. I never really was a fan of that but do like the feature.